modules/pricer/src/main/java/com/opengamma/strata/pricer/credit/cds/PriceType.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.credit.cds;
+
+/**
+ * Enumerates the types of PV that can be returned. 
+ * <p>
+ * The price is usually clean or dirty.
+ */
+public enum PriceType {
+
+  /**
+   * Clean price
+   */
+  CLEAN,
+  /**
+   * Dirty price
+   */
+  DIRTY;
```

Comment:
This concept is not CDS specific. Perhaps it should go in `com.opengamma.strata.pricer.common`.

Is there any more documentation that could be added (that would be valid for bond and CDS)